spatialmath/geometry.go

```diff
@@ -37,7 +37,7 @@ const (
 	SphereType      = GeometryType("sphere")
 	CapsuleType     = GeometryType("capsule")
 	PointType       = GeometryType("point")
-	CollisionBuffer = 1e-8 // objects must be separated by this many mm to not be in collision
+	CollisionBuffer = 1 // objects must be separated by this many mm to not be in collision
```

Comment:
This is a pretty significant change and we should not sign off on this until we see what the impact is to collision checking for more than just the UR. For starters, getting the benchmarking tests running with this change is a must.

Reply:
I agree we'll want to look at the benchmarking, but I doubt this change will actually be terribly impactful to any real-world use case. To allow non-colliding objects to come within one millimeter of one another is to express optimism about the accuracy of the servos and the arm manufacturer's control loops.

Reply:
I think its less about that and more about the fact that we should expect a user to specify geometries for an arm implementation to 3 or 4 decimal places in Cartesian coordinates. Otherwise I agree, this isn't likely to be impactful in our most immediate real-world use cases.